### PR TITLE
Allow users to continue after a eKYC error

### PR DESCRIPTION
**Pull request by prasanthu**
https://trello.com/c/DYgeveyi/235-app-stuck-in-ekyc-rejected-screen

### CODE REVIEW

File: dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift [hunk 1]
```diff
@@ -78,7 +78,45 @@ class SingaporeUserHappyFlowWithScanICStageDeciderTests: XCTestCase {
...
+                    simProfiles: nil,
+                    kycStatusMap: KYCStatusMap(jumio: .REJECTED, myInfo: .PENDING, nricFin: .APPROVED, addressPhone: .APPROVED)
+                )
+            ]
+        )
+
+        XCTAssertEqual(decider.compute(context: context, localContext: localContext), .ohNo(.ekycRejected))
+    }
+
+    func testUserHasCompletedJumioFailedAndClearedLocally() {
```
Comment (sgoodwin) [inline dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift]: This name is unclear. What scenario are you testing?

File: dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift [hunk 2]
```diff
@@ -78,7 +78,45 @@ class SingaporeUserHappyFlowWithScanICStageDeciderTests: XCTestCase {
         
         XCTAssertEqual(decider.compute(context: context, localContext: localContext), .address)
     }
-    
+
+    func testUserHasCompletedJumioFailedCallback() {
```
Comment (sgoodwin) [inline dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift]: This name is unclear. What scenario are you testing?

Comment (prasanthu) [reply dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift]: Forget the whole thing. I am removing this PR

File: dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift (hunk 1 above)
Comment (prasanthu) [reply dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift]: Forget this, I am removing this PR

**Pull request closed without merging**